modular_skyrat/master_files/code/modules/uplink/uplink_items.dm

```diff
@@ -131,57 +35,13 @@
 	restricted_roles = list(JOB_BARTENDER)
 
 //EXPLOSIVES
-/datum/uplink_item/explosives/buzzkill_traitor
-	name = "Buzzkill Grenade Box"
-	desc = "A box with three grenades that release a swarm of angry bees upon activation. These bees indiscriminately attack friend or foe \
-			with random toxins. Courtesy of the BLF and Tiger Cooperative."
-	item = /obj/item/storage/box/syndie_kit/bee_grenades
-	cost = 10
-
-/datum/uplink_item/explosives/viscerators_traitor
-	name = "Viscerator Delivery Grenade"
-	desc = "A unique grenade that deploys a swarm of viscerators upon activation, which will chase down and shred \
-			any non-operatives in the area."
-	item = /obj/item/grenade/spawnergrenade/manhacks
-	cost = 7
-	surplus = 35
-
-/datum/uplink_item/explosives/nukeop_traitor
-	name = "Nuclear Delivery Grenade"
-	desc = "A very confusing grenade containing 2 dehydrated nuclear operatives. Stand back when primed."
-	item = /obj/item/grenade/spawnergrenade/therealnuke
-	cost = 7
-	surplus = 35
```

Comment:
Why?

Reply:
Thought it was a nuke op item because of the naming scheme in the item path. Re-adding now.